invisible_cities/reco/corrections_test.py

```diff
@@ -268,21 +269,38 @@ def test_apply_all_correction_single_maps_raises_exception_when_invalid_map(map_
 
 @few_examples
 @given(float_arrays(size      = 1,
-                   min_value = -198,
-                   max_value = +198),
+                    min_value = -198,
+                    max_value = +198),
+       float_arrays(size      = 1,
+                    min_value = -198,
+                    max_value = +198))
+def test_apply_geo_correction_properly(map_filename, x, y):
+    """
+    The input map is homogeneous, therefore the geometric
+    correction factor must be 1.
+    """
+    maps      = read_maps(map_filename)
+    load_corr = apply_geo_correction(map_e0=maps, norm_strat=NormStrategy.max)
+    corr      = load_corr(x, y)
+    assert_allclose (corr, np.ones_like(corr))
+
+@few_examples
+@given(float_arrays(size      = 1,
+                    min_value = -198,
+                    max_value = +198),
```

Comment:
```suggestion
@given(float_arrays(size=1, min_value=-198, max_value=+198),
```
I think this is short enough to be done in one line, no? If you agree, do it for all 4 lines.